nominees/productive-forests.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Productive Forests",
+  "aliases": [
+    "Productive Forests"
+  ],
+  "description": "Bioverse creates forest inventory to support the sustainable use of non timber related forest products",
+  "website": "https://www.bioverse.io/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/Bioverse-Labs/deep-learning/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 12,
+      "evidenceText": "Productive forests secure economic incentives to maintain ecosystems standing while contributing to food security and nutrition. ",
+      "evidenceURL": "https://www.unicefinnovationfund.org/broadcast/updates/unicef-innovation-fund-bioverse"
+    },
+    {
+      "SDGNumber": 13,
+      "evidenceText": "While productive forests generate income to traditional communities, its protection secure ecosystem services to humanity",
+      "evidenceURL": "https://www.unicefinnovationfund.org/broadcast/updates/unicef-innovation-fund-bioverse"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
```

Comment:
@nathanbaleeta, I think this should this be type 'aimodel'? Perhaps both software and aimodel?

Reply:
Resolved in commit 3222153